kura/examples/org.eclipse.kura.example.eddystone.scanner/OSGI-INF/metatype/org.eclipse.kura.example.eddystone.scanner.EddystoneScannerExample.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<MetaData xmlns="http://www.osgi.org/xmlns/metatype/v1.2.0" localization="en_us">
+    <OCD id="org.eclipse.kura.example.eddystone.scanner.EddystoneScannerExample"
+         name="Eddystone Scanner" 
+         description="Eddystone Scanning Example.">
```

Comment:
"Enable scan for Eddystones. Only UID and URL frame types are supported."
Or should I add more details on Eddystones?